package.xml

```diff
@@ -17,6 +17,12 @@
   <depend>pcre</depend>
   <depend>zlib</depend>
 
+  <test_depend>ament_package</test_depend>
```

Comment:
@Blast545 is this necessary?

Reply:
It was, but ament_cmake_test adds it internally, I will remove it.